src/lib.rs

```diff
@@ -1,3 +1,8 @@
+#![doc = include_str!("../docs/cargo_features.md")]
```

Comment:
This puts the feature list before the rest of the documentation, while it should be after.

<img width="1219" alt="image" src="https://github.com/bevyengine/bevy/assets/59022059/d71d240a-d5fc-476d-b2ae-7424b237c378">

I think just this line needs to be moved back, though. I'm cool with leaving the rest how it is right now.